docs/reference/search/field-caps.asciidoc

```diff
@@ -77,6 +77,15 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailab
   (Optional, Boolean) If `true`, unmapped fields are included in the response.
   Defaults to `false`.
 
+`filters`::
+(Optional, string) Comma-separated list of filters to apply to the response.  The
+following filters are supported: +metadata,-metadata,-parent,-nested,-multifield
+
+`types`::
+(Optional, string) Comma-separated list of field types to include. Any fields that
+do not match one of these types will be excluded from the results.  Defaults to empty,
+meaning that all field types are returned.
```

Comment:
Keyword runtime fields return `KeywordFieldMapper.CONTENT_TYPE` as their family name (which is what is actually checked here).  I also wonder if it's worth adding a `-runtime`, `+runtime` pair of filters. I know you don't like making it too obvious what is runtime and what isn't, but I think it can still be helpful for clients.